src/main/scala/strawman/collection/View.scala

```diff
@@ -9,14 +9,14 @@ import scala.Predef.{<:<, intWrapper}
 trait View[+A] extends Iterable[A] with IterableOps[A, View, View[A]] {
   override def view = this
 
-  def iterableFactory = View
+  def iterableFactory: IterableFactory[View] = View
 
   protected[this] def fromSpecificIterable(coll: Iterable[A]): View[A] = fromIterable(coll)
 
   protected[this] def newSpecificBuilder(): Builder[A, View[A]] =
     immutable.IndexedSeq.newBuilder().mapResult(_.view)
 
-  final protected[this] def coll: this.type = this
+  protected[this] def coll: this.type = this
```

Comment:
Why isn’t that `final` anymore?

Reply:
It needs to be overridden in `SeqView` and `IndexedSeqView` to provide a more narrow return type.